src/GeneratorUtils.php

```diff
@@ -1,22 +1,32 @@
 <?php
 
 namespace CommandString\Utils;
-use InvalidArgumentException;
 
 class GeneratorUtils {
-    public static function uuid(int $length = 16, ?array $characters = null): string
+    public static function uuid(int $length = 16): string
     {
-        $characters = $characters ?? array_merge(range("A", "Z"), range("a", "z"), range(0, 9));
-        $id = "";
+        $salt = random_bytes($length);
+        $time = microtime(true);
+        $hash = hash('sha256', $salt . $time);
+        return $hash;
+    }
```

Comment:
Last I checked hashes return the same length so the parameter is misleading. [How long is the SHA256 hash?](https://stackoverflow.com/questions/2240973/how-long-is-the-sha256-hash/2241014#2241014)

This also breaks backwards compatibility which I want to avoid.

Reply:
> This also breaks backwards compatibility which I want to avoid.
how so?

Reply:
Method performs differently now

Reply:
> Method performs differently now

it does the exact same thing but in a different way. 
`2*3` vs `3*2`

Reply:
Can't supply custom characters now. Also, you never addressed the fact this will always return 256 bits (64 characters).

Reply:
> you never addressed the fact this will always return 256 bits (64 characters).

`random_bytes($length);`

Reply:
You're returning a hash. That hash will **ALWAY** return 256 bits (64 characters) as I mentioned previously. The input does **NOT** affect the output like it does in encryption.